packages/terra-list/src/List.jsx

```diff
@@ -13,41 +12,60 @@ const propTypes = {
    */
   children: PropTypes.node,
   /**
-   * Whether or not the child list items should have a border color applied.
+   * Whether or not the list's child items should have a border color applied.
    */
-  isDivided: PropTypes.bool,
+  dividerStyle: PropTypes.oneOf(['none', 'standard', 'bottom-only']),
+  /**
+   * The padding styling to apply to the child list item content.
+   */
+  paddingStyle: PropTypes.oneOf(['none', 'standard', 'thin']),
```

Comment:
suggesting changing `thin` to `compact` which would loosely would match button api (`isCompact`). curious about others thoughts?

Reply:
updated.